open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb

```diff
@@ -119,19 +119,32 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        if countries_and_region is None or not isinstance(countries_and_region, str) or countries_and_region.strip() == "":
</pre><p>remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/203/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>Done</p>